src/app/models/__tests__/admin_feedback.server.model.spec.ts

```diff
@@ -0,0 +1,66 @@
+import dbHandler from '__tests__/unit/backend/helpers/jest-db'
+import { ObjectId } from 'bson-ext'
+import { omit } from 'lodash'
+import mongoose from 'mongoose'
+
+import { IAdminFeedback } from 'src/types'
+
+import getAdminFeedbackModel from '../admin_feedback.server.model'
+
+const FeedbackModel = getAdminFeedbackModel(mongoose)
+
+describe('form_feedback.server.model', () => {
+  beforeAll(async () => await dbHandler.connect())
+  beforeEach(async () => await dbHandler.clearDatabase())
+  afterAll(async () => await dbHandler.closeDatabase())
+
+  describe('Schema', () => {
+    const DEFAULT_PARAMS: IAdminFeedback = {
+      userId: new ObjectId(),
+      rating: 1,
+      comment: 'feedback comment',
+    }
+
+    it('should create and save successfully', async () => {
+      // Act
+      const actual = await FeedbackModel.create(DEFAULT_PARAMS)
+
+      // Assert
+      expect(actual).toEqual(
+        expect.objectContaining({
+          ...DEFAULT_PARAMS,
+          created: expect.any(Date),
+          lastModified: expect.any(Date),
+        }),
+      )
+    })
+
+    it('should save successfully even when comment param is missing', async () => {
+      // Arrange
+      const paramsWithoutComment = omit(DEFAULT_PARAMS, 'comment')
+      // Act
+      const actual = await FeedbackModel.create(paramsWithoutComment)
+
+      // Assert
+      expect(actual).toEqual(
+        expect.objectContaining({
+          ...paramsWithoutComment,
+          created: expect.any(Date),
+          lastModified: expect.any(Date),
+        }),
+      )
+    })
+
+    it('should throw validation error when rating param is missing', async () => {
+      // Act
+      const actualPromise = new FeedbackModel(
+        omit(DEFAULT_PARAMS, 'rating'),
```

Comment:
Should this be `const paramsWithoutRating = omit(DEFAULT_PARAMS, 'rating')`, following the above convention of `Arranging` before `Act`?

Reply:
Thanks for catching that ken! (clearly I blindly copy pasted from `feedback.server.model.spec.ts` HAHAH [ab89573](https://github.com/opengovsg/FormSG/pull/6454/commits/ab895734fb40ccb94be84a3c90659bde87a801b0)